docs/components/connectors/use-connectors/inbound.md

```diff
@@ -54,17 +59,16 @@ You can still start instances of that process manually via the modeler, which is
 
 ### Modeling the connector message start event
 
-1. Start building your BPMN diagram with an **Event subprocess**.
-2. Add a plain **Message start event (non-interrupting)** into an **Event subprocess**.
-3. Change its template to an inbound connector of your choice (for example, HTTP webhook or a message queue subscription).
-4. Fill in all required properties.
-5. Configure the **Correlation** section if needed
+1. Start building your BPMN diagram with a **Message start event (non-interrupting)**
+2. Change its template to an inbound connector of your choice (for example, HTTP webhook or a message queue subscription).
+3. Fill in all required properties.
+4. Configure the **Correlation** section if needed
```

Comment:
As a first time user, how would I know if this is needed?

Reply:
This is mentioned directly in the element template (only needed for event subprocesses). But I agree it would be nice to mention it here as well